tritium-api/src/main/java/com/palantir/tritium/event/InvocationEventHandler.java

```diff
@@ -54,7 +54,7 @@
     /**
```

Comment:
should we specify `@return` docs for `preInvocation`?

Reply:
Good call. I don't want to recommend returning null from `preInvocation` since our logging all suggests it's symptomatic of an error, and we've never annotated the method as `@Nullable`